.phan/config.php

```diff
@@ -34,10 +34,17 @@
     "unused_variable_detection" => false,
     "suppress_issue_types" => [
         "PhanTypePossiblyInvalidDimOffset",
-        "PhanUndeclaredMethod",
         "PhanTypeMismatchArgument",
         "PhanTypeMismatchProperty",
         "PhanTypeArraySuspiciousNullable",
+        "PhanTypeMismatchReturn",
```

Comment:
We shouldn't be adding to the suppressed types in phan. Every one of these is a bug that's being caught by static analysis. I sent #6404 to revert the change to webdriver. I think after I merge #6404 this PR should be updated to revert that change and then fix this without regressions to our phan rules.